### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by toshipp**
add CONTRIBUTING.md based on CNCF template and modify README.md to adapt to CNCF template.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (toshipp) [inline CONTRIBUTING.md:104]: There are two options DCO and CLA. I choose DCO because it is lightweight.

Please provide your though, maintainers.

Comment (cupnes) [reply CONTRIBUTING.md:104]: I choose DCO too. I think CLA is high operating costs and we don't need as much rigor as the CLA.

File: CONTRIBUTING.md
Comment (cupnes) [inline CONTRIBUTING.md:136]:
```suggestion
- If your code has a breaking change, please update related documents.
```

File: README.md
Comment (cupnes) [inline README.md:25]:
```suggestion
Docker images are available on [ghcr.io](https://github.com/orgs/topolvm/packages).  
```

File: README.md
Comment (cupnes) [inline README.md:69]:
```suggestion
If you have any questions or ideas to contribute, please use [discussions](https://github.com/topolvm/topolvm/discussions).
```

**Review by cupnes — CHANGES_REQUESTED** [review]
Please check and correct them.

File: CONTRIBUTING.md
Comment (llamerada-jp) [inline CONTRIBUTING.md:3]:
```suggestion
* [Contributing Guide](#contributing-guide)
```
Were those generated automatically? Only this line is different from actual title.

File: CONTRIBUTING.md
Comment (llamerada-jp) [inline CONTRIBUTING.md:34]: This one seems to be a different kind. Is it correct?

**Review by llamerada-jp — APPROVED** [review]
Before creating a PR that includes big changes, I would like to discuss it or make a proposal. What do you think?

File: CONTRIBUTING.md
Comment (toshipp) [reply CONTRIBUTING.md:76]: `Following steps` is a subject.

File: CONTRIBUTING.md
Comment (toshipp) [reply CONTRIBUTING.md:34]: It is contained in the template, so I don't know what it means. It is possibly for documents.
But, we have no web site now, so I delete it.

**Comment by toshipp** [comment]
>Before creating a PR that includes big changes, I would like to discuss it or make a proposal. What do you think?

Yeah, it is better. I will improve it.

**Review by llamerada-jp — APPROVED** [review]

File: README.md
Comment (peng225) [inline README.md:43]:
```suggestion
started contributing, please see our [Contributing Guide](CONTRIBUTING.md).
```

**Review by peng225 — CHANGES_REQUESTED** [review]

**Comment by peng225** [comment]
For other reviewers:
https://github.com/cncf/project-template/blob/main/CONTRIBUTING.md

**Review by cupnes — APPROVED** [review]

File: CONTRIBUTING.md
Comment (toshipp) [reply CONTRIBUTING.md:44]: It is a mistake. Thanks.

File: CONTRIBUTING.md
Comment (toshipp) [reply CONTRIBUTING.md:68]: The original template uses both... (e.g. isn't and have not).
I don't know the rule, but change do + not to don't and is + not to isn't.

File: CONTRIBUTING.md
Comment (toshipp) [reply CONTRIBUTING.md:96]: Indeed.

File: CONTRIBUTING.md
Comment (toshipp) [reply CONTRIBUTING.md:67]: fixed.

File: CONTRIBUTING.md
Comment (toshipp) [reply CONTRIBUTING.md:78]: fixed.

File: README.md
Comment (toshipp) [reply README.md:43]: fixed. (it is from the template though :)

File: CONTRIBUTING.md
Comment (toshipp) [reply CONTRIBUTING.md:60]: I interpreted it as that if you have questions to work on the issue, what is it the best way to reach us? I think it is on the original issue.
By contrast, it is preferable to use discussions to ask general questions including how to contribute, IMO.

Comment (peng225) [reply CONTRIBUTING.md:60]: Obeying the CNCF's template, I withdraw this comment.
https://github.com/cncf/project-template/blob/main/CONTRIBUTING.md#ask-for-help

**Review by peng225 — DISMISSED** [review]

**Comment by toshipp** [comment]
I'll squash this after you approve. Please don't merge this.

**Review by daichimukai — CHANGES_REQUESTED** [review]
LGTM. Block merge until squash.

**Comment by daichimukai** [comment]
When squash commits, please "sign off" on commits!

File: README.md
Comment (satoru-takeuchi) [inline README.md:64]:
```suggestion
- [Pod security policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) support is deprecated and will be removed when TopoLVM removes the support of Kubernetes < v1.25. This policy is same as Kubernetes.
```

File: README.md
Comment (satoru-takeuchi) [inline README.md:74]:
```suggestion
[docs](docs/) directory contains the user manual, designs and specifications, and so on.
```

File: README.md
Comment (satoru-takeuchi) [inline README.md:78]:
```suggestion
TopoLVM maintainers presented the motivation and implementation of TopoLVM at KubeCon Europe 2020: https://kccnceu20.sched.com/event/ZerD
```

File: README.md
Comment (toshipp) [reply README.md:16]: done

File: README.md
Comment (toshipp) [reply README.md:26]: done

File: README.md
Comment (toshipp) [reply README.md:42]: done

File: README.md
Comment (toshipp) [reply README.md:69]: done

File: README.md
Comment (toshipp) [reply README.md:64]: done

File: README.md
Comment (toshipp) [reply README.md:74]: done

File: README.md
Comment (toshipp) [reply README.md:78]: done

**Review by satoru-takeuchi — APPROVED** [review]

**Review by cupnes — APPROVED** [review]

**Comment by toshipp** [comment]
squashed. please approve it again.

**Review by cupnes — APPROVED** [review]

**Review by daichimukai — APPROVED** [review]

**Review by peng225 — APPROVED** [review]

**Pull request merged**
